src/themes/Svingle/Message/Message.scss

```diff
@@ -26,5 +26,31 @@ $font-sizes: ( breakpoints.$extra-small: 18px ) !default;
 				margin-bottom: 0;
 			}
 		}
+
+		&-title {
+			max-height: 62px;
+			margin-bottom: 4px !important;
+			text-align: center;
+			font-size: 25px;
```

Comment:
should be `font-size: 1.4em;`,
else the text will always have a fixed size (25px).

If you don't remember this, please read up again on Relative length units:
https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Values_and_units